src/selectors/linesInScope.js

```diff
@@ -0,0 +1,42 @@
+// import { linesInScope } from "../utils/scopes";
+import { getOutOfScopeLocations, getSelectedSource } from "../selectors";
```

Comment:
this is the first selector that spans stores (sources, ast) with that in mind I created a new selectors directory. I'm not sure if that makes sense. What do others think?